pkg/api/staticHandlers.go

```diff
@@ -0,0 +1,170 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// StaticHandlers.go exposes an implementation of MethodHandlers with static (compile time) configs.
+
+package api
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/golang/glog"
+	"github.com/golang/protobuf/ptypes/struct"
+
+	"google.golang.org/genproto/googleapis/rpc/code"
+	"google.golang.org/genproto/googleapis/rpc/status"
+
+	denyadapter "istio.io/mixer/adapter/denyChecker"
```

Comment:
I think this is spot on. One of the reasons I created a new class rather than modify methodHandlers was because it felt weird having //pkg/api depend //adapters. I think I'll update this PR to:
- Use the ubermanager in methodHandlers.go, combining the existing methodHandlers with the changes in staticHandlers.go relating to the uber manager and request dispatch.
- Have the methodHandlers constructor accept a config struct like you described

This should mean a lot less throwaway code and the correct dependencies (//pkg/api doesn't know about //adapters, it's //cmd/server that wires them together).